meilisearch-http/src/main.rs

```diff
@@ -28,8 +28,21 @@ pub fn main() -> Result<(), MainError> {
                         .into(),
                 );
             }
-            env_logger::init();
-        }
+
+            if !opt.no_analytics {
+                let _sentry = sentry::init((
+                    "https://5ddfa22b95f241198be2271aaf028653@sentry.io/3060337",
+                    sentry::ClientOptions {
+                        release: sentry::release_name!(),
+                        ..Default::default()
+                    },
+                ));
+                sentry::integrations::panic::register_panic_handler();
+                sentry::integrations::env_logger::init(None, Default::default());
```

Comment:
Are you sure that it works correctly? I remember that the scope in which you register the panic handler counts.